docs/installation/linux.md

```diff
@@ -58,15 +59,22 @@ it.
 
 DEV requires Redis version 4.0 or higher.
 
-We recommend following Digital Ocean's extensive [How To Install and Configure
-Redis on Ubuntu
-16.04](https://www.digitalocean.com/community/tutorials/how-to-install-and-configure-redis-on-ubuntu-16-04)
+We recommend following Digital Ocean's extensive
+[How To Install and Configure Redis on Ubuntu 16.04](https://www.digitalocean.com/community/tutorials/how-to-install-and-configure-redis-on-ubuntu-16-04)
 (available for other Linux distributions as well) to setup Redis.
 
+### Elasticsearch
+
+DEV requires Elasticsearch version 7 or higher.
+
+We recommend following
```

Comment:
I think we should be super explicit here telling people to install the OSS version of Elastic and not the version with the Elastic license.

The documentation isn't super clear about that.